src/components/books/AddBooks.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../../redux/actions/bookActions';
+
+const AddBooks = () => {
```

Comment:
Add book functionality is not working. Please make sure the user is able to add books. Kindly check the error in the console when the user clicks on Add book button.